.gitignore

```diff
@@ -95,3 +95,8 @@ doc/gapmacrodoc.idx
 /tst/testlibgap/wscreate
 /tst/testlibgap/wsload
 /tst/testlibgap/*.out
+
+# emacs backup and lock files
+*~
+\#*\#
+.\#*
```

Comment:
Strictly speaking, I would argue that such things should go into your private global gitignore list... Alas, I also don't see any harm in adding it to the repository, so ... *shrug*